core/src/com/unciv/ui/newgamescreen/PlayerPickerTable.kt

```diff
@@ -190,4 +207,16 @@ class PlayerPickerTable(val previousScreen: PreviousScreenInterface, var gamePar
         nationsPopup.open()
         update()
     }
-}
\ No newline at end of file
+
+    private fun getAvailablePlayerCivs(): ArrayList<Nation> {
+        var nations = ArrayList<Nation>()
+        for (nation in previousScreen.gameSetupInfo.ruleset.nations.values
+                .filter { !it.isCityState() && it.name != Constants.barbarians }) {
```

Comment:
I think you're looking for .filter{ it.isMajorCiv() }

Reply:
You are right! :)

Reply:
Will be fixed in the next commit.